cmd/root.go

```diff
@@ -44,52 +47,18 @@ in with avalanche subnet create myNewSubnet.`,
 	rootCmd.PersistentFlags().StringVar(&logLevel, "log-level", "ERROR", "log level for the application")
 
 	// add sub commands
-	rootCmd.AddCommand(subnetCmd)
-	rootCmd.AddCommand(networkCmd)
 
-	// add hidden backend command
-	backendCmd.Hidden = true
-	rootCmd.AddCommand(backendCmd)
-
-	// subnet create
-	subnetCmd.AddCommand(createCmd)
-	createCmd.Flags().StringVar(&filename, "file", "", "file path of genesis to use instead of the wizard")
-	createCmd.Flags().BoolVar(&useSubnetEvm, "evm", false, "use the SubnetEVM as the base template")
-	createCmd.Flags().BoolVar(&useCustom, "custom", false, "use a custom VM template")
-	createCmd.Flags().BoolVarP(&forceCreate, forceFlag, "f", false, "overwrite the existing configuration if one exists")
-
-	// subnet delete
-	subnetCmd.AddCommand(deleteCmd)
-
-	// subnet deploy
-	subnetCmd.AddCommand(deployCmd)
-	deployCmd.Flags().BoolVarP(&deployLocal, "local", "l", false, "deploy to a local network")
-
-	// subnet describe
-	subnetCmd.AddCommand(readCmd)
-	readCmd.Flags().BoolVarP(
-		&printGenesisOnly,
-		"genesis",
-		"g",
-		false,
-		"Print the genesis to the console directly instead of the summary",
-	)
+	// We need to pass double pointers to app into these functions since app
```

Comment:
probably this comment is somewhat obsolete

Reply:
removed